tests/task/test_calcZernikesNeuralTask.py

```diff
@@ -0,0 +1,227 @@
+        self.config.alignet_path = '/home/peterma/research/Rubin_AO_ML/training/alignnet_logs/lightning_logs/version_0/checkpoints/best_alignnet_120.ckpt'
+        self.config.aggregatornet_path = '/home/peterma/research/Rubin_AO_ML/training/aggregator_logs/lightning_logs/version_0/checkpoints/best_aggregator.ckpt'
+        self.config.dataset_param_path = '/home/peterma/research/Rubin_AO_ML/TARTS/TARTS/dataset_params.yaml'
+        self.config.device = 'cpu'
+
+        # Initialize the neural task
+        self.task = CalcZernikesNeuralTask(config=self.config, name="Neural Task")
+        
+        # Set the run name for data collection
+        self.runName = "pretest_run_cwfs"
+        
+        # Initialize Butler and registry for data loading
+        self.butler = Butler.from_config(self.repoDir)
+        self.registry = self.butler.registry
+
+        # Define data IDs for extra and intra focal exposures
+        # Using adjacent detectors (191 and 192) from the same visit
+        self.visitNum = 2031063000001
+        self.dataIdExtra = {
+            "instrument": "LSSTCam",
+            "detector": 191,
+            "exposure": self.visitNum,
+        }
+        self.dataIdIntra = {
+            "instrument": "LSSTCam",
+            "detector": 192,
+            "exposure": self.visitNum,
+        }
+
+    def test_load_model(self) -> None:
+        """
+        Test we can load the models
+        """
+        print(self.task)
+
+    def test_load_intra_extra_exposures(self) -> None:
+        """
+        Test loading a pair of intra and extra focal exposures from the Butler repository.
+        
+        This test follows the pattern from test_cutOutDonutsCwfsTask.py to load
+        exposure data and donut catalogs for testing the neural network algorithm.
+        """
+        # Butler and registry are already initialized in setUp()
+        # Data IDs are also already defined in setUp()
+
+        # Load post-ISR exposures from Butler
+      
+        exposureExtra = self.butler.get(
+            "raw", dataId=self.dataIdExtra, collections=["LSSTCam/raw/all"]
+        )
+        exposureIntra = self.butler.get(
+            "raw", dataId=self.dataIdIntra, collections=["LSSTCam/raw/all"]
+        )
+        # exposureExtra = self.butler.get(
+        #     "post_isr_image", dataId=self.dataIdExtra, collections=[self.runName]
+        # )
+        # exposureIntra = self.butler.get(
+        #     "post_isr_image", dataId=self.dataIdIntra, collections=[self.runName]
+        # )
+        
+        # Load camera information
+ 
+        
+        # Verify that we have valid data
+        self.assertIsNotNone(exposureExtra, "Extra focal exposure should not be None")
+        self.assertIsNotNone(exposureIntra, "Intra focal exposure should not be None")
+        
+        # Store the loaded data as instance variables for use in other tests
+        self.exposureExtra = exposureExtra
+        self.exposureIntra = exposureIntra
```

Comment:
What's the purpose of this test? It looks to me as if you do the exact same thing in the next test